tutorials/best_practices/godot_interfaces.rst

```diff
@@ -108,11 +108,13 @@ access.
         };
 
         // Warn users if the value hasn't been set.
-        public String _GetConfigurationWarnings()
+        public String[] _GetConfigurationWarnings()
         {
             if (EnemyScn == null)
-                return "Must initialize property 'EnemyScn'.";
-            return "";
+            {
+                return { "Must initialize property 'EnemyScn'." };
+            }
+            return {};
```

Comment:
Same, but also it'd be better to use `Array.Empty` to reuse empty arrays instead of allocating a new one.

```suggestion
            return Array.Empty<string>();
```